src/compiler/checker.ts

```diff
@@ -10902,7 +10904,7 @@ namespace ts {
             return reportedError;
         }
 
-        function *generateJsxAttributes(node: JsxAttributes): ElaborationIterator {

```

Comment:
Like these are base on MS's own coding standard. I do not exactly know what MS's coding standard is but if this match might be a great fix?

Reply:
I did a global search for function * and function* to see what the preferred style was. I am seeing both however..

Reply:
Well, if that is the case, either we will have to wait for someone to review this or we can just completely ignore this. Anyway both options would not harm the result. :D